include/openenclave/internal/sgx/sgxproperties.h

```diff
@@ -32,7 +32,8 @@ OE_INLINE bool oe_sgx_is_valid_num_tcs(uint64_t x)
 OE_INLINE bool oe_sgx_is_valid_attributes(uint64_t x)
 {
     /* Check for illegal bits */
-    if (x & ~(OE_SGX_FLAGS_DEBUG | OE_SGX_FLAGS_MODE64BIT))
+    if (x & ~(OE_SGX_FLAGS_DEBUG | OE_SGX_FLAGS_MODE64BIT |
+              OE_SGX_FLAGS_PROVISION_KEY | OE_SGX_FLAGS_EINITTOKEN_KEY))
```

Comment:
Should we add support in `OE_SGX_FLAGS_EINITTOKEN_KEY` if it is being deprecated by Intel anyway?

Reply:
@MWShan may correct me if I'm wrong. Intel doesn't deprecate LE or `EINITTOKEN_KEY`. It's the Linux community's choice to support FLC-enabled platforms only, while other OS's may continue to use/support LE. Technically, an SGX enclave is OS-agnostic and IMHO, as an SDK, OE should support all attributes documented in the SDM.